### DISCUSSION TITLE
Base requirements

### DISCUSSION THREAD

**Discussion by codefly** [Cardano-Data-Hub/cardano-chain-analytics-dataset#2] [Ideas]
Here are some basic questions to get us started:

- How is transaction volume increasing over time?
- What are the transaction fees being paid to the network
- How much ada is moving around the network
- How is money consolidating or distributing amongst wallets
- How many people are delegating

**Comment by TravelerJo** [comment]
I would like to know about small SPOs. Say from 30000 ADA staked up to 2 million Ada staked. That are on the current version and available to create blocks.

How many are there?
How many blocks are they actually assigned? By general size of course. 
How many delegators do they have and what is the avg per delegator?

**Reply by codefly** [reply]
Awesome questions! I'll have to investigate if we can determine the version of the node software the operator is running based on the block records.
